.cbmc-batch/jobs/aws_cryptosdk_cmm_generate_enc_materials/Makefile

```diff
@@ -0,0 +1,72 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+#########
+# if Makefile.local exists, use it. This provides a way to override the defaults
+sinclude ../Makefile.local
+#otherwise, use the default values
+include ../Makefile.local_default
+
+include ../Makefile.string
+
+#########
+# Local vars
+
+# gets full coverage, runtime 3 min. 
+MAX_NUM_ITEMS ?= 1
+DEFINES += -DMAX_NUM_ITEMS=$(MAX_NUM_ITEMS)
+
+#########
+ABSTRACTIONS += $(SRCDIR)/c-common-helper-stubs/aws_hash_table_no_slots_override.c
+ABSTRACTIONS += $(SRCDIR)/c-common-helper-stubs/aws_string_destroy_override.c
+ABSTRACTIONS += $(SRCDIR)/c-common-helper-stubs/error.c
+ABSTRACTIONS += $(SRCDIR)/helper-src/openssl/bn_override.c
+ABSTRACTIONS += $(SRCDIR)/helper-src/openssl/ec_override.c
+ABSTRACTIONS += $(SRCDIR)/helper-src/openssl/evp_override.c
+
+# Enable after https://github.com/diffblue/cbmc/issues/5344 is fixed
+#AWS_DEEP_CHECKS = 1
```

Comment:
Do we have an issue here to update the proofs with this comment?

Reply:
https://github.com/aws/aws-encryption-sdk-c/issues/529